admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/utils/SyncMasterDataServiceHelper.java

```diff
@@ -1652,6 +1926,12 @@ private boolean isChangesFound(String entityName, LocalDateTime lastUpdated) {
 			case "DocumentCategory":
 				result = documentCategoryRepository.getMaxCreatedDateTimeMaxUpdatedDateTime();
 				break;
+			case "Gender":
+				result = genderRepository.getMaxCreatedDateTimeMaxUpdatedDateTime();
+				break;
+			case "IndividualType":
+				result = individualTypeRepository.getMaxCreatedDateTimeMaxUpdatedDateTime();
+				break;
```

Comment:
I don't see Device, DeviceSpecification and DeviceType in this switch

Reply:
Added device related cases.